### PR TITLE
chore: improve the look of PoweredByGoogle label of the Autocomplete

### PR DESCRIPTION
**Pull request by mupkoo**
[TACO-1495](https://toptal-core.atlassian.net/browse/TACO-1495)

### Description

Couple of minor improvements to the `PoweredByGoogle` label of the `Autocomplete` component.

- Replace the current image with a high-dpi version. It is still the same image and we go it from their guidelines
- Put the label into the `fixedFooter` of the `SelectOptions`, so it is more or less inline with the limit option of the [`Select`](https://picasso.toptal.net/?path=/story/forms-select--select#limit) component

Those changes has been discussed with Dia Ibram from the design team 🙏 
https://docs.google.com/document/d/1CTj-QzTSOnLyBOCM_s2goS0PAps0dGu89JfpelCAmro/edit?disco=AAAAZ6nJAbU

### How to test

- Spec should be green ✅
- Open the temploy and check the `Autocomplete` component with `Powered by Google` label on -> https://picasso.toptal.net/power-by-google-hdpi/?path=/story/forms-autocomplete--autocomplete#powered-by-google-label
- It should look fabulous ✨

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Shot 2022-05-25 at 13 18 58 | Shot 2022-05-25 at 13 18 42 |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: .changeset/famous-dragons-kick.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': patch
+---
+
+Improve the look of `PoweredByGoogle` label of the `Autocomplete`
```
Comment (mupkoo) [inline .changeset/famous-dragons-kick.md:5]: Should I change this to

```suggestion
---
### Autocomplete

- Improve the look of `PoweredByGoogle` label
```

File: packages/picasso/src/Autocomplete/PoweredByGoogle.tsx [hunk 2]
```diff
@@ -6,11 +7,13 @@ import { Container } from '../Container'
...
...UVuxQAoY7H6/xCcBXQdLKD4NqqiakKmsxG2i1uxwjdOGAsQvAP3u3dmbKt2NsI6Glc4tmlitvJ4NmbZRWM0b30HrEIpG/Gbba3GILxF4lridquo3p+LLnj+hTrHFpev+mlwDgtG/dxrACfAenAf8DDwKWAOdE4iTqA3wDIg4c1IQHD35ignSDGYE6QB/CF4DayWZi8rQo40EEzLgcQBnKnDuyNtAIVAbGvYJHYT5Q/awC+x+yE8GEwLohOY4MU6WBfBlG0OEgGV8qRsbUpKeH8Po760chBg/SPAp4DZNwSKreDXwR+A+Rx8JhL1zbh8iHlZBgGCOuAiVQLGquZaQwQfbnc2gl8DU9+50FUQGuoJ/gxutsUnl/KyyoyarTsUHqg/G0KDNC345Mp8ENuwGvg34I3SawStrIU0k8jzjkRD7BuK+lHeUYHWWxF+Bb9/wx0Cqs6d+KarafiO0dpWw7B9OQnt/Iyv2vG2axkPnPMtr7kfJOOWDBYQ5wCwVD+PH+a/RFo79InLRypH3yhhtGnB6m//mfM/lJK/kWSnfpxNKei6eV1AVy4FASi0kigy0kALNcDt93+ACV60Vh8BR9SKGuCbLu8/f4TWQoAuOFD3ibc9I1LOrZza8dZQ4VaMLGjFsqOiIw2IBr4Gj2zbXpHIyD2yGuCPkRWoY65FqZO01VAJ9/uZaig4+On5Aj6UgSVCa7bNKXgG1OYNiCr8wmuA28SR3lPuxFaUW/mIWkEDAB/caXKvFgzEIfNtvHIQVo376wFa/0LS3ANpnfc/sJTsh+VGFtBhqS/KnIMGhkNGDvZfzvccI4fyI5GUBubAO5fB5CVDvRifrMzFGdLDdqLpfUR2xcXLMpwrTU27hqDs+fivIod1HpZqQn6+yALKT1+RdH4aIPDwPhWJB/rrXF/0p1U0gF88nIfD5AVm4dhaTXDvQcViu5J3gGghpe5AQX67SiSmmXna0h8BUFtq+/9fXbzEyS0YaSWYr+0jakUN4GXfzTxQzqUKvo7Hxcx4RQ4/6p9LeS2Rid6CtURrUZ5cNdAFgrL92gd/BEC5au4w5fhRLoBoCrZbZcFvvmj14Nh5Xnu76P7Abx4dZq35ZxcAKvWyroeb6a5N/qVHOSINRBr4XDXAH1rD264B+IHo7vhBZiwCenvYt26fVyP/D6pKYxO86572AAAAAElFTkSuQmCC'
```
Comment (mupkoo) [inline packages/picasso/src/Autocomplete/PoweredByGoogle.tsx:13]: This is the high-dpi version of the image that was downloaded from the Google guidelines
https://developers.google.com/maps/documentation/places/web-service/policies#powered

**Comment by mupkoo** [comment]
@toptal-bot run all

File: .changeset/famous-dragons-kick.md (hunk 1 above)
Comment (augustobmoura) [reply .changeset/famous-dragons-kick.md:5]: Yes, changesets should follow the [template guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md)

File: packages/picasso/src/Autocomplete/PoweredByGoogle.tsx (hunk 2 above)
Comment (augustobmoura) [reply packages/picasso/src/Autocomplete/PoweredByGoogle.tsx:13]: Hmm, this image is getting quite big for inlining actually. I will see which options we have for this

File: .changeset/famous-dragons-kick.md (hunk 1 above)
Comment (mupkoo) [reply .changeset/famous-dragons-kick.md]: Updated!

File: packages/picasso/src/Autocomplete/PoweredByGoogle.tsx (hunk 2 above)
Comment (mupkoo) [reply packages/picasso/src/Autocomplete/PoweredByGoogle.tsx:13]: I ran it though [TinyPNG](https://tinypng.com) and the size is around `2.5kb`, but yes, it is quite bigger than then previous one

**Review by TomasSlama — APPROVED** [review]

**Review by OndrejTuma — APPROVED** [review]

**Review by dmaklygin — APPROVED** [review]

File: packages/picasso/src/Autocomplete/PoweredByGoogle.tsx (hunk 2 above)
Comment (mupkoo) [reply packages/picasso/src/Autocomplete/PoweredByGoogle.tsx:13]: Hey @augustobmoura should I wait for this before merging?

Comment (augustobmoura) [reply packages/picasso/src/Autocomplete/PoweredByGoogle.tsx:13]: Yeah, we don't have too many options unfortunately, so this will need to cut it for now

**Review by augustobmoura — APPROVED** [review]
Thanks for the contribution, @mupkoo! :+1:

**Pull request merged**